### PR TITLE
Link community resources

### PR DESCRIPTION
**Pull request by EmilFattakhov**
This PR adds a new page with community-build tools. 
It uses a couple of Iframes to the community build projects to increase awareness, promote them, and to make navigation simpler. 

To view, check out the `/docs/participate/community_resources/community-build-tools` page

### CODE REVIEW

File: docs/participate/community_resources/community-build-tools.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Community Build Tools
```
Comment (jim-counter) [inline docs/participate/community_resources/community-build-tools.md:2]:
```suggestion
title: Community Built Tools
```

File: docs/participate/community_resources/community-build-tools.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+sidebar_position: 2
+description: essential info to file systems. partitioning, and mounting.
+keywords:
+    - Subspace Network
+    - Community build tools
```
Comment (jim-counter) [inline docs/participate/community_resources/community-build-tools.md:7]:
```suggestion
    - Community built tools
```

File: docs/participate/community_resources/community-build-tools.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+### Community-maintained list of userful, third-party tools
```
Comment (jim-counter) [inline docs/participate/community_resources/community-build-tools.md:10]:
```suggestion
### Community-maintained list of useful, third-party tools
```

File: docs/participate/community_resources/community-build-tools.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+[Wolfrage76's](https://github.com/wolfrage76) **list of third-party tools**.
```
Comment (jim-counter) [inline docs/participate/community_resources/community-build-tools.md:12]:
```suggestion
[Wolfrage76](https://github.com/wolfrage76)'s **list of third-party tools**. 
```

File: docs/participate/community_resources/community-build-tools.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+description: essential info to file systems. partitioning, and mounting.
+keywords:
+    - Subspace Network
+    - Community build tools
+---
+
+### Community-maintained list of userful, third-party tools
+
+[Wolfrage76's](https://github.com/wolfrage76) **list of third-party tools**. 
+- Link to the [Website.](https://subspace.ifhya.com/)
```
Comment (jim-counter) [inline docs/participate/community_resources/community-build-tools.md:13]:
```suggestion
- Link to the [website](https://subspace.ifhya.com/).
```

File: docs/participate/community_resources/community-build-tools.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+- Contact **Wolfrage** in the Subspace Discord to add your tool.
+
+<iframe src="https://subspace.ifhya.com/" width="100%" height="650px" frameborder="0" scrolling="auto"></iframe>
+
+### Plotting speed measurement tool
+
+[Hakehardware's](https://github.com/hakehardware) **Plot Report** that allows you to measure your plotting speed and adds it to the table for comparison with other hardware plotting speeds.
```
Comment (jim-counter) [inline docs/participate/community_resources/community-build-tools.md:20]:
```suggestion
[Hakehardware](https://github.com/hakehardware)'s **Plot Report** that allows you to measure your plotting speed and adds it to a table for comparison with other hardware plotting speeds. 
```

File: docs/participate/community_resources/community-build-tools.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+- Link to the [Website.](https://subspace.ifhya.com/)
+- Contact **Wolfrage** in the Subspace Discord to add your tool.
+
+<iframe src="https://subspace.ifhya.com/" width="100%" height="650px" frameborder="0" scrolling="auto"></iframe>
+
+### Plotting speed measurement tool
+
+[Hakehardware's](https://github.com/hakehardware) **Plot Report** that allows you to measure your plotting speed and adds it to the table for comparison with other hardware plotting speeds. 
+- Link to [GitHub](https://github.com/hakehardware/plotreport)
+- Link to the [Website](https://plotreport.hakedev.com/)
```
Comment (jim-counter) [inline docs/participate/community_resources/community-build-tools.md:22]:
```suggestion
- Link to the [website](https://plotreport.hakedev.com/).
```

File: docs/participate/community_resources/community-build-tools.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+- Link to the [Youtube channel](https://www.youtube.com/channel/UCakvG7QQp4oL0Rtpiei1yKg)
```
Comment (jim-counter) [inline docs/participate/community_resources/community-build-tools.md:23]:
```suggestion
- Link to the [YouTube channel](https://www.youtube.com/channel/UCakvG7QQp4oL0Rtpiei1yKg).
```

File: versioned_docs/version-latest/participate/community_resources/community-build-tools.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
+---
+title: Community Build Tools
```
Comment (jim-counter) [inline versioned_docs/version-latest/participate/community_resources/community-build-tools.md:2]:
```suggestion
title: Community Built Tools
```

File: versioned_docs/version-latest/participate/community_resources/community-build-tools.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+sidebar_position: 2
+description: essential info to file systems. partitioning, and mounting.
```
Comment (jim-counter) [inline versioned_docs/version-latest/participate/community_resources/community-build-tools.md:4]: Should this be updated? I'm not sure where it's used?

File: docs/participate/community_resources/community-build-tools.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
+---
+title: Community Build Tools
+sidebar_position: 2
+description: essential info to file systems. partitioning, and mounting.
```
Comment (jim-counter) [inline docs/participate/community_resources/community-build-tools.md:4]: Should this be updated? I'm not sure where it's used?

File: versioned_docs/version-latest/participate/community_resources/community-build-tools.md [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+keywords:
+    - Subspace Network
+    - Community build tools
```
Comment (jim-counter) [inline versioned_docs/version-latest/participate/community_resources/community-build-tools.md:7]:
```suggestion
    - Community built tools
```

File: versioned_docs/version-latest/participate/community_resources/community-build-tools.md [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+### Community-maintained list of userful, third-party tools
+
+[Wolfrage76's](https://github.com/wolfrage76) **list of third-party tools**.
```
Comment (jim-counter) [inline versioned_docs/version-latest/participate/community_resources/community-build-tools.md:12]:
```suggestion
[Wolfrage76](https://github.com/wolfrage76)'s **list of third-party tools**. 
```

File: versioned_docs/version-latest/participate/community_resources/community-build-tools.md [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+description: essential info to file systems. partitioning, and mounting.
+keywords:
+    - Subspace Network
+    - Community build tools
+---
+
+### Community-maintained list of userful, third-party tools
+
+[Wolfrage76's](https://github.com/wolfrage76) **list of third-party tools**. 
+- Link to the [Website.](https://subspace.ifhya.com/)
```
Comment (jim-counter) [inline versioned_docs/version-latest/participate/community_resources/community-build-tools.md:13]:
```suggestion
- Link to the [website](https://subspace.ifhya.com/).
```

File: versioned_docs/version-latest/participate/community_resources/community-build-tools.md [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+- Contact **Wolfrage** in the Subspace Discord to add your tool.
+
+<iframe src="https://subspace.ifhya.com/" width="100%" height="650px" frameborder="0" scrolling="auto"></iframe>
+
+### Plotting speed measurement tool
+
+[Hakehardware's](https://github.com/hakehardware) **Plot Report** that allows you to measure your plotting speed and adds it to the table for comparison with other hardware plotting speeds.
```
Comment (jim-counter) [inline versioned_docs/version-latest/participate/community_resources/community-build-tools.md:20]:
```suggestion
[Hakehardware](https://github.com/hakehardware)'s **Plot Report** that allows you to measure your plotting speed and adds it to a table for comparison with other hardware plotting speeds. 
```

File: versioned_docs/version-latest/participate/community_resources/community-build-tools.md [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+[Wolfrage76's](https://github.com/wolfrage76) **list of third-party tools**. 
+- Link to the [Website.](https://subspace.ifhya.com/)
+- Contact **Wolfrage** in the Subspace Discord to add your tool.
+
+<iframe src="https://subspace.ifhya.com/" width="100%" height="650px" frameborder="0" scrolling="auto"></iframe>
+
+### Plotting speed measurement tool
+
+[Hakehardware's](https://github.com/hakehardware) **Plot Report** that allows you to measure your plotting speed and adds it to the table for comparison with other hardware plotting speeds. 
+- Link to [GitHub](https://github.com/hakehardware/plotreport)
```
Comment (jim-counter) [inline versioned_docs/version-latest/participate/community_resources/community-build-tools.md:21]:
```suggestion
- Link to [GitHub](https://github.com/hakehardware/plotreport).
```

File: versioned_docs/version-latest/participate/community_resources/community-build-tools.md [hunk 17]
```diff
@@ -0,0 +1,25 @@
...
+- Link to the [Website](https://plotreport.hakedev.com/)
```
Comment (jim-counter) [inline versioned_docs/version-latest/participate/community_resources/community-build-tools.md:22]:
```suggestion
- Link to the [website](https://plotreport.hakedev.com/).
```

File: versioned_docs/version-latest/participate/community_resources/community-build-tools.md [hunk 18]
```diff
@@ -0,0 +1,25 @@
...
+- Link to the [Youtube channel](https://www.youtube.com/channel/UCakvG7QQp4oL0Rtpiei1yKg)
```
Comment (jim-counter) [inline versioned_docs/version-latest/participate/community_resources/community-build-tools.md:23]:
```suggestion
- Link to the [YouTube channel](https://www.youtube.com/channel/UCakvG7QQp4oL0Rtpiei1yKg).
```

File: docs/participate/community_resources/community-build-tools.md [hunk 19]
```diff
@@ -0,0 +1,25 @@
...
+[Wolfrage76's](https://github.com/wolfrage76) **list of third-party tools**. 
+- Link to the [Website.](https://subspace.ifhya.com/)
+- Contact **Wolfrage** in the Subspace Discord to add your tool.
+
+<iframe src="https://subspace.ifhya.com/" width="100%" height="650px" frameborder="0" scrolling="auto"></iframe>
+
+### Plotting speed measurement tool
+
+[Hakehardware's](https://github.com/hakehardware) **Plot Report** that allows you to measure your plotting speed and adds it to the table for comparison with other hardware plotting speeds. 
+- Link to [GitHub](https://github.com/hakehardware/plotreport)
```
Comment (jim-counter) [inline docs/participate/community_resources/community-build-tools.md:21]:
```suggestion
- Link to [GitHub](https://github.com/hakehardware/plotreport).
```

**Review by jim-counter — CHANGES_REQUESTED** [review]
As we have to rename the file `build` -> `built` should we consider renaming consistently to `community-maintained-tools` everywhere?

File: docs/participate/community_resources/community-build-tools.md (hunk 11 above)
Comment (EmilFattakhov) [reply docs/participate/community_resources/community-build-tools.md:4]: This is used for indexing, it's better to update it. Thanks for catching it!

**Comment by EmilFattakhov** [comment]
Thanks Jim, all comments were addressed now!

**Review by jim-counter — APPROVED** [review]

**Pull request merged**